doc/src/base/punctuation.md

```diff
@@ -22,6 +22,7 @@ Extended documentation for mathematical symbols & functions is [here](@ref math-
 | `'`         | a trailing apostrophe is the [`adjoint`](@ref) (that is, the complex transpose) operator Aᴴ |
 | [`*`](@ref) | the asterisk is used for multiplication, including matrix multiplication and [string concatenation](@ref man-concatenation) |
 | [`/`](@ref) | forward slash divides the argument on its left by the one on its right                      |
+| [`//`](@ref) | double forward slash divides two integers or rational numbers                              |
```

Comment:
```suggestion
| [`//`](@ref) | double forward slash performs exact, rational division                                     |
```
To clarify what makes `//` different from `/`. (e.g. `/` also divides integers and rationals)